src/documents/management/commands/document_consumer.py

```diff
@@ -122,8 +125,8 @@ def _consume(filepath: str) -> None:
     try:
         logger.info(f"Adding {filepath} to the task queue.")
         consume_file.delay(
-            filepath,
-            override_tag_ids=list(tag_ids) if tag_ids else None,
+            ConsumeDocument(DocumentSource.ConsumeFolder, filepath).as_dict(),
```

Comment:
I'd always use named arguments with data classes. Just in case you adjust / reorder attributes in the data class later on.